File: frontend/src/app/@core/modelos/pedido-aux.ts

import { ProductoAux } from "./Varios/producto-aux";


export class PedidoAux {
  constructor() {
    this.codPedido=0;
    this.estadoEntrega="ACEPTADO";
    this.fechaPedido=new Date();
    this.numeroPedido="";
    this.montoTotal=0;
    this.codCliente=0;
    this.codClienteDireccionEnvio=0;
    this.codTipoEnvio=0;
    this.lstProductos=[];
  }

  codPedido:number;
  estadoEntrega:string;
  fechaPedido:Date;
  numeroPedido:string;
  montoTotal:number;
  codCliente:number;
  codClienteDireccionEnvio:number;
  codTipoEnvio:number;
  lstProductos: ProductoAux[];

}

File: backend/Models/Varios/PedidoAux.cs
using System;
using System.Collections.Generic;

namespace backend.Models
{
    public partial class PedidoAux
    {
        public int CodPedido { get; set; }
        public DateTime FechaPedido { get; set; }
        public String NumeroPedido { get; set; }
        public decimal MontoTotal { get; set; }
        public String EstadoEntrega { get; set; }

        public int CodCliente { get; set; }
        public int CodClienteDireccionEnvio { get; set; }
        public int CodTipoEnvio { get; set; }
        public IEnumerable<ProductoAux> lstProductos { get; set; }
    }
}